scripts/tick-cluster.js

```diff
@@ -146,6 +147,48 @@ function statsAll() {
     });
 }
 
+function grid() {
```

Comment:
'grid mode' should be separated out as its own PR. It's best to keep unrelated features separate so that it's easier to revert (if need be) and review.